### PR TITLE
[ArtistVC] Fixed bug in past shows iPad layout

### LINKED ISSUE #258: [Artist][Shows] Past shows list on iPad only shows first 3 shows
I didn't catch this when I was looking for issues with the RN upgrade:

simulator screen shot aug 26 2016 2 45 10 pm

The current release still uses the previous RN version, so it's not a problem for existing users. I'm going to fix this by updating the `MediumShows` component so it uses a column system like the `RelatedArtists` one.

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Fixes #258 

simulator screen shot aug 26 2016 6 13 40 pm

### CODE REVIEW

**Comment by ArtsyMobile** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="true">No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="true">The following files do not have tests: <a href="https://github.com/artsy/emission/blob/17d53b3c06b6ff4d2f94442ece54c0586f78e843/lib/components/artist/shows/medium_list.js">lib/components/artist/shows/medium_list.js</a> &amp; <a href="https://github.com/artsy/emission/blob/17d53b3c06b6ff4d2f94442ece54c0586f78e843/lib/components/artist/shows/metadata.js">lib/components/artist/shows/metadata.js</a></td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by sarahscott** [comment]
Not sure if we need a changelog entry here since this is a bug fix for a bug that doesn't exist in the current release.

**Comment by orta** [comment]
You could add something to the `Dangerfile.js` that allows you to say that it is bug fix ( maybe if you include "#bugfix" in the `github.pr_body`? ) - like we do on eigen for "#trivial"

**Comment by sarahscott** [comment]
I'll make a ticket for that. For now, I have to complete some paperwork and sign off for the night.

Giphy

File: lib/components/artist/shows/large_list.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ class LargeList extends React.Component {
     const window = Dimensions.get('window')
     const isPad = window.width > 700
     const imageWidth = (isPad ? ((window.width - 100) / 2) : (window.width - 40))
-    const imageHeight = Math.floor(imageWidth / 1.5)
+    const imageHeight = Math.floor(imageWidth / 1.6)
```
Comment (alloy) [inline lib/components/artist/shows/large_list.js]: Such magic numbers ✨

**Comment by alloy** [comment]
Ace, good catch and fix 👏

File: lib/components/artist/shows/large_list.js (hunk 1 above)
Comment (sarahscott) [inline lib/components/artist/shows/large_list.js]: It'd probably be more legible if I made an `aspectRatio` variable.

**Pull request merged**